docs/tasks/administer-cluster/weave-network-policy.md

```diff
@@ -105,6 +105,59 @@ spec:
 
 {% endcapture %}
 
+## How do I test if it is working?
+
+### Check weave pod
+```shell
+kubectl get po -n kube-system -o wide
+```
+Check weave working:
+```
+NAME                                    READY     STATUS    RESTARTS   AGE       IP              NODE
+weave-net-1t1qg                         2/2       Running   0          9d        192.168.2.10    workndoe3
+weave-net-231d7                         2/2       Running   1          7d        10.2.0.17       worknodegpu
+weave-net-7nmwt                         2/2       Running   3          9d        192.168.2.131   masternode
+weave-net-pmw8w                         2/2       Running   0          9d        192.168.2.216   worknode2
+```
+
+Each Node has a weave Pod.And all of these Pods are Running and 2/2 READY(each has weave and weave-npc).
+
+
+### Create Networkpolicy and check logs
+After creating a NetworkPolicy, use kubectl to check the logs
+```shell
+kubectl logs -f  weave-net-pmw8w weave-npc  -n kube-system
+```
+Log output looks like this
+```log
+INFO: 2017/08/14 02:22:32.511992 EVENT AddNetworkPolicy {"metadata":{"name":"aaa","namespace":"myns","selfLink":"/apis/extensions/v1beta1/namespaces/myns/networkpolicies/aaa","uid":"67b229fd-8097-11e7-92f3-005056a3bc75","resourceVersion":"1507955","generation":1,"creationTimestamp":"2017-08-14T02:22:22Z"},"spec":{"podSelector":{"matchExpressions":[{"key":"inns","operator":"In","values":["yes"]}]},"ingress":[{"from":[{"podSelector":{"matchExpressions":[{"key":"inns","operator":"In","values":["yes"]}]}}]}]}}
+INFO: 2017/08/14 02:22:32.512103 creating ipset: &npc.selectorSpec{key:"inns in (yes)", selector:labels.internalSelector{labels.Requirement{key:"inns", operator:"in", strValues:[]string{"yes"}}}, ipsetType:"hash:ip", ipsetName:"weave-[T]a=ETzaKA{o*muaFe:2IX(t"}
+INFO: 2017/08/14 02:22:32.538003 adding rule: [-m set --match-set weave-[T]a=ETzaKA{o*muaFe:2IX(t src -m set --match-set weave-[T]a=ETzaKA{o*muaFe:2IX(t dst -j ACCEPT]
+^[^C
+```
+Finally,check iptables rules,logs like that
+```iptables
```

Comment:
This doesn't work to mark up the output properly. Again, please check the netlify build and modify to get the right output.